### ISSUE TITLE
[BLES00016] - Call of Duty 3 - Game No longer Boots

### ISSUE THREAD

**Issue by web1018** [RPCS3/rpcs3#7863]
## Quick summary
On the latest Master version, every time when trying to boot up the game, the game immediately hangs/ Freezes on a black screen with an error - **F {PPU[0x1000000] Thread (main_thread) [0x00029b8c]} VM: Access violation reading location 0x0 (unmapped memory)**. The emulator then stops working and closes.

## Details
RPCS3 v0.0.9 - 10023 - 158e34f Alpha | Hardware Specs :
Nvidia GeForce GTX 960M | Intel Core i7 6700HQ | 16GB of Ram | Windows 10

The issue began from Master version v0.0.9-10021-150d1bcd (#7845). Checking Master Version v0.0.9-10018 and earlier, the game boots up just fine without any issues.

## Screenshoots
![Cod3-crash](https://user-images.githubusercontent.com/51851411/77650340-bea0fb80-6f41-11ea-85e7-14bafd5cfb18.PNG)

## Log
**Cannot get the log file as the entire emulator freezes/crashes and the log file doesn't get populated/made.**

- Thanks to #TesseracT (5316) For support with testing and obtaining the log
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/4387713/RPCS3.log)

## Update

- Checking Master Version v0.0.9 - 10118 the issue with crashing/freezing no longer occurs when booting. I am closing the issue down, unless it starts happening again.

**Comment by elad335** [comment]
Can you send the updated log then? also there's no access violation in old log.

**Comment by web1018** [comment]
Here is the updated log from a latest master version v0.0.9 - 10054. When I tried to get the log from Master version v0.0.9-10021 where the Access violation error began, I just could not get the log because the entire game and emulator would stop working and crash when I try to boot it. I got help from TesseracT who managed to get the log for me. If you check the Master version v0.0.9-10021 you will see access violation error. 

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/4396500/RPCS3.log.gz)

In the current master v0.0.9 - 10054 I can boot up the game just fine, go ingame, but then get a different error while trying to compile the shaders. 

![Capture12](https://user-images.githubusercontent.com/51851411/77812022-e644b080-7074-11ea-8422-fcd221f44431.PNG)
